### PR TITLE
Components: prepare changelogs for release

### PR DESCRIPTION
**Pull request by psealock**
Prepare changelogs for release

### CODE REVIEW

**Comment by jeffstieler** [comment]
Just as as heads up https://github.com/woocommerce/woocommerce-admin/pull/2801 touches some components as well.

**Comment by psealock** [comment]
Thanks for the heads up @jeffstieler, I already made the release though.

The changes in #2801 are particularly useful for WCPay but I can easily make another release. cc @allendav

File: packages/navigation/CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# 3.0.0 (unreleased)
+# 3.0.0
 
 - `getHistory` updated to reflect path parameters in url query.
```
Comment (joshuatf) [inline packages/navigation/CHANGELOG.md]: Could you add the following in here:

```
- Added a new `<Form />` component.
- Stepper component: Add new `content` and 	`description` props.
```

Comment (psealock) [reply packages/navigation/CHANGELOG.md]: Done in https://github.com/woocommerce/woocommerce-admin/pull/2802/commits/68f76aacad0de1dbebbed651f4984aaea63bfe3b

**Pull request merged**